docs/src/main/paradox/user/lang/go/gettingstarted.md

```diff
@@ -0,0 +1,78 @@
+# Getting started with stateful services in Go
+
+## Prerequisites
+
+Go version
+: Cloudstate Go support requires at least Go $cloudstate.go.version$
+
+Build tool
+: Cloudstate does not require any particular build tool, you can select your own.
+
+protoc
+: Since Cloudstate is based on gRPC, you need a protoc compiler to compile gRPC protobuf descriptors. This can be done manually through the [Protocol Buffer Compiler project](https://github.com/protocolbuffers/protobuf#protocol-compiler-installation). 
+
+docker
+: Cloudstate runs in Kubernetes with [Docker](https://www.docker.com), hence you will need Docker to build a container that you can deploy to Kubernetes. Most popular build tools have plugins that assist in building Docker images.
+
+In addition to the above, you will need to install the Cloudstate Go support library by issuing `go get -u github.com/cloudstateio/go-support` or with Go module support let the dependency be downloaded by `go [build|run|test]`. 
+
+By using the Go module support your go.mod file will reference the latest version of the support library or you can define which version you like to use.
+
+go get
+: @@@vars
+```text
+go get -u github.com/cloudstateio/go-support
+```
+@@@
+
+import path
+: @@@vars
+```text
+import "github.com/cloudstateio/go-support"
+```
+@@@
+
+go.mod
+: @@@vars
+```
+module example.com/yourpackage
+    require (
+        github.com/cloudstateio/go-support $cloudstate.go-support.version$
+    )
+go $cloudstate.go.version$
+```
+@@@
+
+## Protobuf files
+
+The Cloudstate Go Support Library provides no dedicated tool beside the protoc compiler to build your protobuf files. The Cloudstate protocol protobuf files as well as the shopping cart example application protobuf files are provided by the Cloudstate Repository.
+
+In addition to the protoc compiler, the gRPC Go plugin is needed to compile the protobuf file to *.pb.go files. Please follow the instructions at the [Go support for Protocol Buffers](https://github.com/golang/protobuf) project page to install the protoc compiler as well as the `protoc-gen-go` plugin which also includes the Google standard protobuf types.
+
+To build the example shopping cart application shown earlier in @ref:[gRPC descriptors](../../features/grpc.md), you could simply paste that protobuf into `protos/shoppingcart.proto`. You may wish to also define the Go package using the `go_package` proto option, to ensure the package name used conforms to Go package naming conventions
+
+```proto
+option go_package = "example/shoppingcart";
+```
+
+Now if you place your protobuf files under protobuf/ and run `protoc --go_out=. --proto_path=protobuf ./protobuf/*.proto`, you'll find your generated protobuf files in `example/shoppingcart`.
+
+## Creating and starting a server
```

Comment:
todo?

Reply:
- [ ] remove it. starting the server is described in the section below.